whitelabelco/sofo-foods
# Cp/feat/become customer form change

### Description 💻

Resolves: #176 

### Implementation 💡

Client requested changes to Become a Customer form -- see ticket for required fields

#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- click "Become a Customer"
- verify form title and form inputs match the change request
- open console and complete form -- verify formData is logging correctly

### Screenshots/Demos 📷

### Checklist ✅
